install_custom_module_map/after/Example.xcodeproj

```diff
@@ -53,7 +53,7 @@ Targets:
         Shell Path: "/bin/sh"
         Shell Script: '"${PODS_ROOT}/Target Support Files/Pods-Example/Pods-Example-frameworks.sh"
 
-          '
```

Comment:
I think I've seen this before, not sure whats wrong or you need a rebase?

Reply:
I tried rebasing this branch and my CocoaPods branch but no luck :( will keep digging